src/index.html

```diff
@@ -11,7 +11,156 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application`"
+      method="post"
+      name="test-form"
+    >
```

Comment:
write all attributes from new line
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application`"
      method="post"
      name="test-form"
    >
```